packages/fluentui/docs/project.json

```diff
@@ -10,6 +10,10 @@
         {
           "projects": ["react-northstar"],
           "target": "build:info"
+        },
+        {
+          "projects": ["react-portal-compat-context"],
```

Comment:
docs uses webpack path aliases, but it also depends on v9 package which cannot be used as alias (https://github.com/microsoft/fluentui/blob/master/scripts/gulp/src/config.ts#L12), thus we orchestrate physical build of this cross framework package